docs/twig-components/method/title.md

```diff
@@ -2,5 +2,16 @@
 
 `title(locale = "", length = 120)` is a Twig filter to ...
```

Comment:
is a Twig filter to ... should be removed and keep the description below or continue after the name of the function